### ISSUE TITLE
Investigate which pbench cli tools require privileged containers

### ISSUE THREAD

**Issue by akrzos** [openshift-scale/workloads#51]
Some of the pbench tools require privileged, however some may not, we should understand what machine data is available from pbench without privileged containers.

**Comment by akrzos** [comment]
mpstat, iostat, disk tools can report data back
sar looks to be working, however I believe is only collecting network stats on the containers visible network interface (HostNetwork is actually what exposes all network interfaces to be collected on for sar)
pidstat will only be able to see the processes within the container and not the additional host processes/containers
perf tool requires privileged containers

So in short we seem to lose pidstat, perf and some sar data.
